ERROR - 2023-03-27 07:59:03 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\PSBF\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\PSBF\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'db_surat', 3306, '', 0)
#1 C:\xampp\htdocs\PSBF\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\PSBF\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\PSBF\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\PSBF\app\Models\M_Login.php(13): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\PSBF\app\Controllers\Home.php(40): App\Models\M_Login->validateLogin('admin', 'admin')
#6 C:\xampp\htdocs\PSBF\system\CodeIgniter.php(934): App\Controllers\Home->validasi_login()
#7 C:\xampp\htdocs\PSBF\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\PSBF\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\PSBF\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\PSBF\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\PSBF\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\PSBF\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\PSBF\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\PSBF\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\PSBF\app\Models\M_Login.php(13): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\PSBF\app\Controllers\Home.php(40): App\Models\M_Login->validateLogin('admin', 'admin')
#5 C:\xampp\htdocs\PSBF\system\CodeIgniter.php(934): App\Controllers\Home->validasi_login()
#6 C:\xampp\htdocs\PSBF\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\PSBF\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\PSBF\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\PSBF\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2023-03-27 07:59:03 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tb_user`
WHERE `username` = :username:
AND `password` = :password:', [...], false)
 3 APPPATH\Models\M_Login.php(13): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Home.php(40): App\Models\M_Login->validateLogin('admin', 'admin')
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->validasi_login()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\PSBF\\public\\index.php')
INFO - 2023-03-27 07:59:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-03-27 07:59:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-03-27 08:05:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-03-27 08:05:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-03-27 08:51:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-03-27 08:52:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-03-27 08:52:18 --> mysqli_sql_exception: Table 'db_surat.tb_arsip_surat' doesn't exist in C:\xampp\htdocs\PSBF\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\PSBF\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\PSBF\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\PSBF\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\PSBF\system\Database\BaseBuilder.php(1700): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\PSBF\app\Models\M_Admin.php(23): CodeIgniter\Database\BaseBuilder->countAllResults()
#5 C:\xampp\htdocs\PSBF\app\Controllers\Home.php(14): App\Models\M_Admin->count_data()
#6 C:\xampp\htdocs\PSBF\system\CodeIgniter.php(934): App\Controllers\Home->index()
#7 C:\xampp\htdocs\PSBF\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\PSBF\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\PSBF\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\PSBF\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-03-27 08:52:18 --> Table 'db_surat.tb_arsip_surat' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `tb_arsip_surat`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `tb_arsip_surat`')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `tb_arsip_surat`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1700): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `tb_arsip_surat`', [], false)
 5 APPPATH\Models\M_Admin.php(23): CodeIgniter\Database\BaseBuilder->countAllResults()
 6 APPPATH\Controllers\Home.php(14): App\Models\M_Admin->count_data()
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\PSBF\\public\\index.php')
INFO - 2023-03-27 08:52:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-03-27 08:52:46 --> mysqli_sql_exception: Table 'db_surat.tb_arsip_surat' doesn't exist in C:\xampp\htdocs\PSBF\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\PSBF\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\PSBF\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\PSBF\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\PSBF\system\Database\BaseBuilder.php(1700): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\PSBF\app\Models\M_Admin.php(23): CodeIgniter\Database\BaseBuilder->countAllResults()
#5 C:\xampp\htdocs\PSBF\app\Controllers\Home.php(14): App\Models\M_Admin->count_data()
#6 C:\xampp\htdocs\PSBF\system\CodeIgniter.php(934): App\Controllers\Home->index()
#7 C:\xampp\htdocs\PSBF\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\PSBF\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\PSBF\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\PSBF\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-03-27 08:52:46 --> Table 'db_surat.tb_arsip_surat' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `tb_arsip_surat`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `tb_arsip_surat`')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `tb_arsip_surat`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1700): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `tb_arsip_surat`', [], false)
 5 APPPATH\Models\M_Admin.php(23): CodeIgniter\Database\BaseBuilder->countAllResults()
 6 APPPATH\Controllers\Home.php(14): App\Models\M_Admin->count_data()
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\PSBF\\public\\index.php')
CRITICAL - 2023-03-27 08:54:04 --> Undefined variable $count_skeluar
in APPPATH\Views\admin.php on line 497.
 1 APPPATH\Views\admin.php(497): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $count_skeluar', 'C:\\xampp\\htdocs\\PSBF\\app\\Views\\admin.php', 497)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\PSBF\\app\\Views\\admin.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('admin', [], true)
 5 APPPATH\Controllers\Home.php(22): view('admin', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\PSBF\\public\\index.php')
INFO - 2023-03-27 09:03:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-03-27 09:04:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2023-03-27 09:09:24 --> You must set the database table to be used with your query.
in SYSTEMPATH\Database\BaseConnection.php on line 882.
 1 SYSTEMPATH\Model.php(642): CodeIgniter\Database\BaseConnection->table(null)
 2 SYSTEMPATH\Model.php(832): CodeIgniter\Model->builder()
 3 APPPATH\Controllers\Home.php(14): CodeIgniter\Model->__call('find_all', [])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\PSBF\\public\\index.php')
CRITICAL - 2023-03-27 09:12:03 --> Undefined property: stdClass::$nik
in APPPATH\Views\admin.php on line 566.
 1 APPPATH\Views\admin.php(566): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$nik', 'C:\\xampp\\htdocs\\PSBF\\app\\Views\\admin.php', 566)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\PSBF\\app\\Views\\admin.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('admin', [], true)
 5 APPPATH\Controllers\Home.php(22): view('admin', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\PSBF\\public\\index.php')
INFO - 2023-03-27 09:15:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
